tomcat/manifest.json

```
@@ -31,11 +31,13 @@
       "spec": "assets/configuration/spec.yaml"
     },
     "monitors": {},
-    "dashboards": {},
+    "dashboards": {
+      "Tomcat Overview": "assets/dashboards/overview.json"
```

```suggestion
      "tomcat--overview": "assets/dashboards/overview.json"
```

This key needs to match the old short name for the dashboard.